core/templates/dev/head/domain/dashboard/DashboardBackendApiService.js

```diff
@@ -33,7 +33,26 @@ oppia.factory('DashboardBackendApiService', [
     });
   };
 
+  var _fetchExplorationStats = function(explorationId) {
+    var requestUrl = (
+      '/dashboardhandler/explorationstats/?exp_id=' + explorationId);
+    return $q(function(resolve, reject) {
+      $http.get(requestUrl, {
+        cache: true
+      }).then(function(response) {
+        if (resolve) {
```

Comment:
I don't think the `if` on `resolve` and `reject` are necessary. The `$q` library should always pass in these args, no?